app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/43b06fc24f1ddcb819de2e20dacc4fe82bd270b7/app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml#L2"></span>The resource <code>R.mipmap.ic_launcher_round</code> appears to be unused</td>
    </tr>
  </tbody>
</table>